src/templates/semantic/wizard/form.hbs

```diff
@@ -0,0 +1,38 @@
+<div style="position: relative;">
```

Comment:
Same here with the styles.

Reply:
@randallknutson this seems to be what's currently in bootstrap and bootstrap 3 of the current 4.x codebase.